mainpage.dox

```diff
@@ -2,7 +2,7 @@
 \mainpage
 \htmlinclude manifest.html
 
-\b tictactoe 
+\b baxter_tictactoe 
 
 <!-- 
 Provide an overview of your package.
```

Comment:
what does this file do? Is it useful? I put the comment to remember us in sight of #8